src/Randomizer.Shared/Models/TrackerDungeonState.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Randomizer.Shared.Models {
+
+    public class TrackerDungeonState
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        public int TrackerStateId { get; set; }
+        public string Name { get; set; }
+        public bool Cleared { get; set; }
+        public int RemainingTreasure { get; set; }
+        public int Reward { get; set; }
```

Comment:
I'll have to review the usage of the Shared project in the SMZ3 project. I initially tried to do that, but it didn't let me due to the SMZ3 project not being a dependency, and I can't introduce it as a dependency as it would create a cyclic dependency. But I think with all of this being managed by the App I can probably pull out all references to the Shared project from the SMZ3 one now.

Reply:
Yeah, some of the basic enums that are used everywhere could probably be moved to there